test/common/TestBed.cpp

```diff
@@ -220,17 +220,20 @@ namespace RcclUnitTesting
     }
   }
 
-  void TestBed::ExecuteCollectives()
+  void TestBed::ExecuteCollectives(std::vector<int> const &currentRanks)
   {
     int const cmd = TestBedChild::CHILD_EXECUTE_COLL;
-    ++TestBed::NumTestsRun();
-
+    if (currentRanks.empty() || (currentRanks.size() > 1)) ++TestBed::NumTestsRun(); // this wont be accurate on the case of multiple children running send or receive
```

Comment:
On second thought, we can just leave it as is and treat each ncclSend / ncclRecv as a test run.
This counter isn't terribly important